### PR TITLE
updated 2.4 nano rx PCBs

### PR DESCRIPTION
**Pull request by JyeSmith**
- Lager XO footprint for the FL520WFMT1.
- Remove ground from beneath the UFL.

### CODE REVIEW

File: PCB/2400MHz/RX_Nano_Ceramic/README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+
+v1.1 PCB
+- Lager XO footprint for the FL520WFMT1.
```
Comment (wvarty) [inline PCB/2400MHz/RX_Nano_Ceramic/README.md:3]: "Larger"?

File: PCB/2400MHz/RX_Nano_Diversity/README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+
+v1.1 PCB
+- Lager XO footprint for the FL520WFMT1.
```
Comment (wvarty) [inline PCB/2400MHz/RX_Nano_Diversity/README.md:3]: Making me want beer man :|

File: PCB/2400MHz/RX_Nano/README.md [hunk 3]
```diff
@@ -5,6 +5,10 @@ v1.1 PCB
 - Lager IPEX pads
 - Added boot jumper pads under the button and base of the PCB.  These can be used in place of the button.  Any solder paste applied to these pads via a stencil will need to be removed before adding a button.
 
+v1.2 PCB
+- Lager XO footprint for the FL520WFMT1.
```
Comment (wvarty) [inline PCB/2400MHz/RX_Nano/README.md:9]: Here too

File: PCB/2400MHz/RX_Nano/README.md [hunk 4]
```diff
@@ -5,6 +5,10 @@ v1.1 PCB
...
+- Remove ground form beneath the UFL.
```
Comment (maybenikhil) [inline PCB/2400MHz/RX_Nano/README.md:10]: from

File: PCB/2400MHz/RX_Nano_Ceramic/README.md (hunk 1 above)
Comment (JyeSmith) [reply PCB/2400MHz/RX_Nano_Ceramic/README.md:3]: Don't mind if I do.

**Pull request merged**